.travis.yml

```diff
@@ -11,7 +11,13 @@ install:
 
 services:
   - mongodb
-# make mongodb a replica set
+ddons:
```

Comment:
You should start copying my stuff: https://github.com/alcaeus/mongo-php-adapter/blob/master/.travis.yml :stuck_out_tongue: